test.js

```diff
@@ -618,6 +618,32 @@ var PLURAL_TESTS = [
 ];
 
 /**
+ * Is plural tests.
+ *
+ * @type {Array}
+ */
+var IS_PLURAL_TESTS = [
+  ['whisky', false],
+  ['echos', true],
+  ['bluffs', true],
+  ['incentive', false],
+  ['sleeve', false]
+];
+
+/**
+ * Is singular tests.
+ *
+ * @type {Array}
+ */
+var IS_SINGULAR_TESTS = [
+  ['whisky', true],
+  ['echos', false],
+  ['bluffs', false],
+  ['incentive', true],
+  ['sleeve', true]
+];
```

Comment:
Couldn't these just use the existing test suite since the left is always singular and the right is always the plural?